docs/user_guide/wrappers/Wrapper.rst

```diff
@@ -116,6 +116,56 @@ to select only a subset of the variables.
     X_train_t = selector.transform(X_train.fillna(0))
     X_test_t = selector.transform(X_test.fillna(0))
 
+Even though Feature Engine has its own implementation of OneHotEncoder, you may want to use Scikit-Learn Implementation in order to access different options, such as drop first Category.
```

Comment:
Could you replace this sentence by the below:

Even though Feature-engine has its own implementation of OneHotEncoder, you may want to use Scikit-Learn'stransformer in order to access different options, such as drop first Category. In the following example, we show you how to apply Scikit-learn's OneHotEncoder to a subset of categories using the SklearnTransformerWrapper.